homeassistant/components/homeassistant/strings.json

```diff
@@ -11,6 +11,10 @@
     "python_version": {
       "title": "Support for Python {current_python_version} is being removed",
       "description": "Support for running Home Assistant in the current used Python version {current_python_version} is deprecated and will be removed in Home Assistant {breaks_in_ha_version}. Please upgrade Python to {required_python_version} to prevent your Home Assistant instance from breaking."
+    },
+    "integration_key_no_support": {
+      "title": "Integration key YAML not supported",
```

Comment:
The current message is hard to grasp. The logged error is already better. But probably we can do even better? 

CC @c0ffeeca7